### PR TITLE
Devops 300

### PR DESCRIPTION
**Pull request by cwsatpolymath**
* Add source code for utility binaries
* Add `polymesh` main and sidecar container templates
* Add `polymesh` helm chart
* Update docker-compose files

### CODE REVIEW

**Comment by cwsatpolymath** [comment]
@Tamir-Polymath @ashneverdawn or @JesseALancaster when you have time please review containers/**/*.go (2 files).

**Review by ashneverdawn — CHANGES_REQUESTED** [review]
Instead of using `go get` to fetch dependencies, you'll want versioned dependencies using go mod. If you run `go mod init your_project_name` in the directory where your `package main` files are, Go will generate a `go.mod` file for you. Then if you run `go mod tidy` (or build/run your project), it will generate a `go.sum` file. You should commit both of these (for each go project)

You will have to set the appropriate versions of your dependencies in your `go.mod`
Go doesn't automatically detect major versions above 2, so `jsonrpc2` will be a bit finicky. In this case you'll want: `github.com/AdamSLevy/jsonrpc2/v14 v14.0.0` in your `go.mod` files. Go might get confused with `github.com/AdamSLevy/jsonrpc2 v2.0.0-incompatible` If it does, also update the import in your go code to: `jsonrpc2 "github.com/AdamSLevy/jsonrpc2/v14"` and you won't have any issue.

Also, this means you won't need to use `go get` manually in your container since dependencies will be fetched automatically when the project is built.

File: containers/polymesh-peer-updater/src/updater.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    "fmt"
+    "os"
+    "github.com/AdamSLevy/jsonrpc2"
+    "log"
+    "github.com/fsnotify/fsnotify"
+    "bufio"
+)
+
+func addReservedPeer(peer string) {
+    fmt.Println("Adding peer %s", peer )
```
Comment (ashneverdawn) [inline containers/polymesh-peer-updater/src/updater.go:13]: `%s` is a formatting directive, but `fmt.Println` doesn't interpret those. It just separates its arguments with spaces.

So you probably want either:
`fmt.Printf("Adding peer %s\n", peer)` 
or
`fmt.Println("Adding peer", peer)`

**Review by ashneverdawn — CHANGES_REQUESTED** [review]

**Comment by ashneverdawn** [comment]
If you wish, you can use a linter that's more advanced than Go's builtin one.
For example, if I use this one:
`golangci-lint run -E gocyclo -E golint -E gofmt`
It gives me these:
```
health-check.go:22:19: SA1012: do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use (staticcheck)
        err := c.Request(nil, "http://localhost:9933", "system_health", params, &r)
```
```
updater.go:19:19: SA1012: do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use (staticcheck)
        err := c.Request(nil, "http://localhost:9933", "system_addReservedPeer", params, &result)
```

**Review by ashneverdawn — APPROVED** [review]

**Review by quinndiggitypolymath — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1,2 @@
 /        @polymathnetwork/Ops
+*        @polymathnetwork/Ops
```
Comment (maxsam4) [inline .github/CODEOWNERS:2]: The old definition is made redundant by the new definition.

**Review by maxsam4 — APPROVED** [review]

**Review by satyamakgec — APPROVED** [review]

**Pull request merged**